src/styles/colors.js

```diff
@@ -1,13 +1,13 @@
 const colors = {
-  colorPrimary: '#3f51b5',
+  colorPrimary: '#A6F3D6',
   colorPrimaryLight: '#757de8',
-  colorPrimaryDark: '#002984',
+  colorPrimaryDark: '#07377A',
   colorSecondary: '#f2f4ff',
   colorSecondaryLight: '#ffffff',
   colorSecondaryDark: '#bfc1cc',
   colorAccent: '#46d3f2',
   colorTextPrimary: '#212121',
```

Comment:
Observando o figma podemos observar que os textos só podem ser de duas cores. Azul ou branco. Sendo a maioria do texto do app escrito em azul. Logo a cor de texto primeira deve ser azul e a cor de texto secundaria deve ser branco. 
```suggestion
  colorTextPrimary: '#07377A',
```

Reply:
Ok, já está alterado.